ehrapy/plot/supervised/_feature_importances.py

```diff
@@ -0,0 +1,30 @@
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+from anndata import AnnData
+
+
+def feature_importances(adata: AnnData, key: str = "feature_importances", n_features: int = 10):
+    """
+    Plot features with greates absolute importances as a barplot.
+
+    Args:
+        adata: :class:`~anndata.AnnData` object storing the data. A key in adata.var should contain the feature
+            importances, calculated beforehand.
+        key: The key in adata.var to use for feature importances. Defaults to 'feature_importances'.
+        n_features: The number of features to plot. Defaults to 10.
+
+    Returns:
```

Comment:
We remove the `Returns:` section if the function does not return anything. I even asked chatGPT to confirm whether this is fine :)

> 
In Google style docstrings, if a function does not return anything (i.e., it implicitly returns `None`), it is considered good practice to omit the `Returns` section entirely. The absence of a `Returns` section implicitly indicates that the function does not return a value or that its return value is not significant. Including a `Returns` section that states `None` is not wrong, but it is unnecessary and can be seen as redundant. Therefore, for brevity and clarity, you should omit the `Returns` section in such cases.